### DISCUSSION TITLE
코드블럭 폰트 변경 건의 + 오류

### DISCUSSION THREAD

**Discussion by jhhan128** [pronist/hELLO#93] [Ideas]
코드블럭 폰트가 일반 고딕체로 나옵니다.
이를 고정폭 글꼴로 바꾸는 것은 어떨까요?

한번 제안드려봅니다.

또한 제가 html에서 건드린 게 없음에도 Designed by...  가 나오지 않습니다. 
이것도 버그인 것 같네요.

**Comment by pronist** [comment]
1. 이것은 고려해보도록 하겠습니다.
2. 가로 사이즈 **1600px** 이하에서는 designed by 가 출력되지 않습니다. 해당 스킨의 반응형의 기준은 1600px, 1200px, ~ 320px 입니다.

**Comment by pronist** [comment]
@jhhan128 코드블럭 폰트를 고정폭 글꼴인 **D2Coding** 으로 바꾸었습니다. 
건의해주셔서 감사합니다!
